crates/daphne-server/tests/e2e/test_runner.rs

```diff
@@ -437,114 +438,7 @@ impl TestRunner {
         }
 
         let resp = client
```

Comment:
Why are we deleting all of this?

Reply:
I refactored to make `leader_put_expect_ok` and `leader_post_expect_ok` one function that takes a http::Method parameter, to simplify testing the backwards compatibility of the upload report functions.